### ISSUE TITLE
Support for debuging realm database 

### ISSUE THREAD

**Issue by vivekpanchal** [amitshekhariitbhu/Android-Debug-Database#166]
Is there any way i could debug realm database with this library because i dont see my realm database in browser. I can only see shared preferences .

**Comment by amitshekhariitbhu** [comment]
Currently, this doesn't support realm.

**Comment by vivekpanchal** [comment]
will it be in the future ?

**Comment by amitshekhariitbhu** [comment]
Yes. It would be great if anyone wants to contribute. PRs are most welcome.

**Comment by dotrinhdev** [comment]
I also care about this feature.
